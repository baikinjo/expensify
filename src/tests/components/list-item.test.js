import React from 'react'
import { shallow } from 'enzyme'

import expenses from '../fixtures/expenses'
import ListItem from '../../components/list-item'

test('should render list item correctly', () => {
  const wrapper = shallow(<ListItem {...expenses[0]} />)
  expect(wrapper).toMatchSnapshot()
})
