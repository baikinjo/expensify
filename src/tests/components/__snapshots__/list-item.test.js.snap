// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render list item correctly 1`] = `
<Link
  className="list-item"
  to="/edit/1"
>
  <div>
    <h3
      className="list-item__title"
    >
      gum
    </h3>
    <span
      className="list-item__sub-title"
    >
      Dec 31, 1969
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $12.03
  </h3>
</Link>
`;
